package.json

```diff
@@ -58,12 +59,12 @@
     "lint": "npm-run-all --parallel lint:js lint:sass",
     "lint:js": "eslint src",
     "lint:sass": "sass-lint --verbose --no-exit src/**/*.scss",
-    "test": "npm run build && web-ext run -s dist",
     "watch:js": "onchange -p -v 'src/**/*.js' -- npm run bundle:js",
     "watch:css": "onchange -p -v 'src/**/*.scss' -- npm run bundle:css",
     "watch:assets": "onchange -p -v 'src/{manifest.json,icons/*,popup/*.html}' -- npm run copy:assets",
     "storybook": "start-storybook -s ./dist -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "test": "npm run lint && echo \"tests TBD\""
```

Comment:
Oh, so, turns out for testing stuff on browser restart, I need the package, because the temporary add-on won't be there next time, so I think I'm going to add a call to `npm run build` to the package step, if that sounds good to you.